dtmi/azurertos/devkit/gsgmicrochipsame54-1.json

```diff
@@ -0,0 +1,73 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:azurertos:devkit:gsgmicrochipsame54;1",    
+    "@type": "Interface",
+    "displayName": "Microchip SAM E54 Getting Started Guide",
+    "description": "Example model for the Azure RTOS Microchip SAM E54 Getting Started Guide",
+    "contents": [
+        {
+            "@type": [
+                "Telemetry",
+                "Temperature"
+            ],
+            "name": "temperature",
+            "displayName": "Temperature",
+            "unit": "degreeCelsius",
+            "schema": "double"
+        },
+        {
+            "@type": [
+                "Telemetry",
+                "RelativeHumidity"
+            ],
+            "name": "humidity",
+            "displayName": "Humidity",
+            "unit": "percent",
+            "schema": "double"
+        },
+        {
+            "@type": [
```

Comment:
could add semantic type

Reply:
Both the humity and pressure already are sematic types. Was this comment meant for a different line?